### PR TITLE
Simplify load sample data

### PR DESCRIPTION
**Pull request by sarayourfriend**
## Fixes

I set up Openverse from scratch today and ran into a very strange issue with `load_sample_data.sh` that I cannot track down the precise cause of. The version on `main` hangs completely, without any feedback, and apparently passes malformed SQL commands to `psql`.

## Description

The changes in this PR get the script running as expected for me again. They also, I believe, somewhat simplify things, through the introduction of two patterns:

1. The new `exec_sql` function which not only DRYs up the code a bit, but also uses what is apparently a more reliable approach for passing a set of SQL commands to `psql`. The same approach is used for interacting with Redis as well.
2. Taking advantage of bash's built-in multiline string support, rather than using docstring style. This is easier to read and keep track of (IMO) and has less potential variations. It's also easier to assign to variables or pass as arguments to bash functions.

## Testing Instructions

CI should work, all API testing steps rely on this script. You should also be able to run it locally.

I really don't know _why_ the script as-is didn't work on this machine. The only thing I can think of is maybe the bash version is different?

```
$ bash --version
GNU bash, version 5.2.15(1)-release (x86_64-pc-linux-gnu)
```

## Checklist

- [x] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [N/A] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.
- [N/A] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by dhruvkb** [comment]
The changes look fine to me, and the CI tests continue to pass, which means there isn't anything logically wrong here. But the more interesting thing to me here is why the previous version of the script did not work on your machine if it worked everywhere else (like CI, for example).

The Mac users on the team are on very old versions of Bash so, if that script worked for us, it should've worked on all newer versions.

```
❯ bash --version
GNU bash, version 3.2.57(1)-release (arm64-apple-darwin23)
Copyright (C) 2007 Free Software Foundation, Inc
```

File: load_sample_data.sh [hunk 1]
```diff
@@ -56,35 +58,35 @@ fi
...
+usernames = ['continuous_integration', 'deploy']
+for username in usernames:
+	if User.objects.filter(username=username).exists():
+		print(f'User {username} already exists')
+		continue
+	if username == 'deploy':
+		user = User.objects.create_superuser(username, f'{username}@example.com', 'deploy')
+	else:
+		user = User.objects.create_user(username, f'{username}@example.com', 'deploy')
+	user.save()
```
Comment (dhruvkb) [inline load_sample_data.sh:75]: The Python part here should be indented with 4 spaces, instead of tabs, as per PEP-8.

**Review by dhruvkb — APPROVED** [review]
LGTM, except for a stylistic change and a bigger question around why the script did not work before these refactors.

**Review by krysal — APPROVED** [review]
Works locally and the simplication of multiline strings is quite nice!

**Pull request merged**